WordPress/Classes/ViewRelated/Menus/MenuItemSourceFooterView.m

```diff
@@ -57,8 +58,16 @@ - (void)setupSourceCell
     cell.frame = self.bounds;
     cell.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
     cell.alpha = 0.0;
-    [cell setTitle:@"Dummy Text For Sizing the Label"];
     [self addSubview:cell];
+    _spinner = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+    UIWindow *window = [[UIApplication sharedApplication] keyWindow];
+    UIView *topView = window.rootViewController.view;
+    CGFloat xPlace = topView.frame.size.width;
+    CGFloat yPlace = cell.frame.size.height;
+    CGFloat heightOfIndicator = cell.frame.size.height - 4;
+    CGFloat widthOfIndicator = heightOfIndicator;
+    _spinner.frame = CGRectMake(xPlace / 2 - widthOfIndicator / 2 , yPlace / 2 - heightOfIndicator / 2, widthOfIndicator, heightOfIndicator);
+    [self addSubview:_spinner];
```

Comment:
Could we add this setup to its own method?

Another suggestion:
We could use Autolayout to center the spinner to its superview. In this way we can simplify this snippet :)